object.js

```diff
@@ -0,0 +1,145 @@
+const data = {
+  projects: [
+    {
+      key: 0,
+      title: "Multi-Post Stories ",
+      image: "./image/Snapshoot_portfolio.png",
```

Comment:
- [ ] The images are still broken, this is because the name here doesn't match the name of the image in the folder /image
it's `./image/Snapshoot_Portfolio.png`, with the Portfolio, not portfolio!

-Kindly consider making any file's name in lower case to avoid such small mistakes in the future 😄_